assets/sass/_variables.scss

```diff
@@ -1,30 +1,30 @@
 // Page Properties
-$fonts: 'montserrat', Arial, Helvetica, sans-serif;
+$fonts: 'proxima nova', 'montserrat', Arial, Helvetica, sans-serif;
 $default-border-radius: 3px;
 $container-width: 400px;
 $container-min-width: 300px;
 $num-tabs: 5;
 
 // Font Size
-$font-size: 13px;
-$font-size-small: 12px;
-$font-size-header: 14px;
-$font-size-footer: 11px;
-$font-size-default: 14px;
+$font-size: 14px;
+$font-size-small: 13px;
+$font-size-header: 15px;
+$font-size-footer: 12px;
+$font-size-default: 15px;
 
 // Font Weight
 $font-weight-body: normal;
 $font-weight-labels: 600;
-$font-weight-header: bold;
+$font-weight-header: 600;
 
 // Theme Color
 $primary-bg-color: #fff;
 $secondary-bg-color: #f9f9f9;
 
 // Text Color
```

Comment:
- Looked at history of this file, `header-text-color`, `primary-button-bg-color` and `google-icon-bg-color` are recent changes (within a year). shall we keep them instead of override by loginpage variables?

Reply:
That probably makes sense. I could be wrong but I don't think we show the google signin buttons on our standard login page. I think you need to enable a custom signin page in order to login with IDP. Maybe @edburyenegren-okta can confirm?

Reply:
yeah, 3rd-party IdP is for custom signin page as today.

Reply:
OK, I have reverted the google icon styles

Reply:
what about other 2 change set as I mentioned earlier.

Reply:
Thanks, didn't catch that earlier. I've reverted those two changes as well. This may cause very a slight change in appearance on the Okta loginpage when we remove those variables in okta-core. As long as we are using the correct/intentional value I think it is good that we make it consistent.